cv32e40x/regress/cv32e40x_obi_bus_err.yaml

```diff
@@ -12,7 +12,7 @@ builds:
     dir: cv32e40x/sim/uvmt
 
 tests:
-  hello-world:
+  corev_rand_instr_obi_err:
```

Comment:
Might want to change the description on this one as well.

Reply:
Thanks.  This has been addressed.